protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -171,6 +171,10 @@ func genValidateFunc(g *protogen.GeneratedFile, msg *protogen.Message) {
 		hasRepeatedExt := hasExt && f.Desc.Cardinality() == protoreflect.Repeated
 		isMessageField := f.Desc.Kind() == protoreflect.MessageKind
 
+		// - don't generate for loop for not an array of msg because it wil be handled differently
+		// - only generatae for loop if there are validations for each single elments otherwise
+		shouldGenerateFor := hasRepeatedExt && !isMessageField && hasNonRepeatedValidationExtensions(f)
```

Comment:
```suggestion
		// - don't generate for loop for not an array of msg because it will be handled differently
		// - only generate for loop if there are validations for each single elements otherwise
```

- Can we better structure this comment? 
- If we rename `shouldGenerateFor` to `shouldRangeOverField`, will that better capture the intent of what the variable controls?